src/main/resources/hudson/plugins/sshslaves/SSHConnector/help-tcpNoDelay.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    Enable/Disables the TCP_NODELAY flag on the SSH connection.
+    If set, disable the Nagle algorithm. This means that segments are always sent as soon as possible,
+    even if there is only a small amount of data. When not set,
+    data is buffered until there is a sufficient amount to send out,
```

Comment:
"data" is plural, hence why I said "data are buffered" ;)

Reply:
For any non-native speakers, there is controversy over the correct usage: https://en.wikipedia.org/wiki/Data_(word). I personally consider it an uncountable noun and prefer `data is` :)

Reply:
this text is from the man page, so It is not my fault ;)

Reply:
All good! :)